docs/templates/creating.md

```diff
@@ -0,0 +1,59 @@
+#  Creating, editing, and updating templates
+
+You create and edit Coder templates as [Terraform](./concepts.md)
+configuration files (`.tf`) and any supporting files, like a README or
+configuration files for other services.
+
+## Who creates templates?
+
+The [Template Admin](../admin/users.md) role (and above) can create
+templates. End users, like developers, create workspaces from them.
+
+Templates can also be [managed with git](./change-management.md),
+allowing any developer to propose changes to a template.
+
+You can give different users and groups access to templates with
+[role-based access control](../admin/rbac.md).
+
```

Comment:
```suggestion
The [Template Admin](../admin/users.md) role (and above) can create
templates. End users, like developers, create workspaces from them. You can give different users and groups access to templates with [role-based access control](../admin/rbac.md).

Templates can also be [managed with git](./change-management.md),
allowing any developer to propose changes to a template.
```

IMO this flows a bit better. Looking at the docs rendering, it is a bit ackward for these to all be on seperate lines as well.